docs/features/fiat-rates.md

```diff
@@ -4,23 +4,23 @@ Suite provides several types of fiat rates:
 
 -   Current fiat rate: Used on Dashboard, next to the account balance, for converting a crypto amount in send form to a fiat currency, etc...
 -   Weekly rates: In addition to current rate we also fetch 7 days old rate and based on the difference we either show green or red arrow next to an exchange rate (can be seen in assets table on Dashboard).
--   Historical fiat rate: Exchange rate at the time of facilitating a transaction. Used in list of transactions to calculate daily deltas and in transaction detail modal. Not supported for ERC20 tokens.
+-   Historical fiat rate: Exchange rate at the time of facilitating a transaction. Used in list of transactions to calculate daily deltas and in transaction detail modal.
 
 ## Providers
 
--   Blockbook: For all main networks (BTC, LTC, ETH) except XRP and ERC20 tokens
+-   Blockbook: For all main networks (BTC, LTC, ETH + ERC-20 tokens, ...) except XRP and ADA
```

Comment:
```suggestion
-   Blockbook: For all main networks (BTC, LTC, ETH + ERC-20 tokens, ...) except XRP, ADA and SOL
```

Reply:
It's better to fix those small things using fixup feature of git, it doesn't really makes sense to have commit like [chore(docs): update coins fetched from blockbook](https://github.com/trezor/trezor-suite/pull/10385/commits/9d72819bb25efa6da0849bb41b9837eba6b4440f)